system/datatypes/peer_reviewed_article.php

```diff
@@ -1158,14 +1158,25 @@ private function createDiscussionCommentForm(DocPage $discussion, array $data =
 	 */
 	private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$comment = __('art-comment');
+		$react = __('art-discussion');
 		$new = $discussion === null;
 		$commentFieldName = self::FIELD_NAME_DISCUSSION_COMMENT . '/' . ($new ? 'new_' . $type : $discussion->getId());
+		if ($new) {
 		$html = <<<EOF
 			<div class="new-comment $type">
 			<div>
 				<strong><label for="$commentFieldName"> $comment </label></strong><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
 			</div>
 EOF;
+		}
+		else {
+			$html = <<<EOF
+			<div class="new-comment $type">
+			<div>
+				<strong><label for="$commentFieldName"> $react </label></strong><textarea name="$commentFieldName" id="$commentFieldName" cols="36" rows="4"></textarea>
+			</div>
```

Comment:
Please leave away the `rows` and `columns` and the `<strong>`  in these statements, beter to instead give the fields a descriptive class, so styling can be adapted via hypha.css.

Alternative:
`<label for="$commentFieldName" class="comment-field-name-label"> $react </label><textarea name="$commentFieldName" id="$commentFieldName" class="comment-field-name"></textarea>`

Reply:
This is the subject of #217 and fixed by #219. I think it is out of scope for this PR, so I'm resolving this conversation.